lib/Button.js

```diff
@@ -147,99 +168,107 @@ export default class Button extends Component {
         const textStyle = (() => {
             if (disabled || !(overrides && overrides.textColor)) {
                 return textStyleMap[shape][theme][type];
+            } else {
```

Comment:
because of this, I'd recommend [excluding whitespace changes when viewing this diff](https://github.com/react-native-material-design/react-native-material-design/pull/94/files?w=1).